pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -5768,3 +5768,18 @@ SRID=4326;POINT (359 90)
 SRID=4326;POINT (-179 -23)
 SRID=4326;LINESTRING (2 2, 5 -60, -160 0)
 SRID=4326;POLYGON ((354 -10, 6 -10, 0 20, 354 -10), (3 2, 359 -1, 1 -5, 3 2))
+
+
+query T
+SELECT ST_AsText(
+    ST_Snap(poly,line, ST_Distance(poly, line)*1.25)
+  ) AS polysnapped
+FROM (SELECT
+  ST_GeomFromText('MULTIPOLYGON(
+    (( 26 125, 26 200, 126 200, 126 125, 26 125 ),
+      ( 51 150, 101 150, 76 175, 51 150 )),
+      (( 151 100, 151 200, 176 175, 151 100 )))') As poly,
+       ST_GeomFromText('LINESTRING (5 107, 54 84, 101 100)') As line
+	) As _;
```

Comment:
i tihnk this `AS _` should bre replaced by `tbl (poly, line)` to make it easier to parse.
or use `SELECT ST_asText(poly, ...) FROM [SELECT ST_GeomFromText(...)]`

Reply:
would this work?
```
query T
SELECT ST_AsText(
    ST_Snap(poly,line, ST_Distance(poly, line)*1.25)
) AS polysnapped
FROM (SELECT
    ST_GeomFromText('MULTIPOLYGON(
      (( 26 125, 26 200, 126 200, 126 125, 26 125 ),
      ( 51 150, 101 150, 76 175, 51 150 )),
      (( 151 100, 151 200, 176 175, 151 100 )))') As poly,
    ST_GeomFromText('LINESTRING (5 107, 54 84, 101 100)') As line
  ) tbl(poly, line);
```

Reply:
If it passes ya